src/EnergyPlus/BoilerSteam.hh

```diff
@@ -171,15 +164,12 @@ namespace BoilerSteam {
 
 struct BoilerSteamData : BaseGlobalStruct
 {
-    int numBoilers = 0;
     bool getSteamBoilerInput = true;
     Array1D<BoilerSteam::BoilerSpecs> Boiler;
```

Comment:
There is no reason for a 1D array of anything that isn't `int`, `Real64`, or `bool` to be an `Array1D` rather than an EPVector.